package.json

```diff
@@ -5,7 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha --compilers js:babel-register --recursive tests/unit/setup-node.js tests/unit/ --require ignore-styles",
+    "test-acceptance:generate-report": "mocha --compilers js:babel-register tests/acceptance/setup/test.js -R index.js",
+    "test-acceptance": "mocha --compilers js:babel-register tests/acceptance/test.js",
```

Comment:
Let's refactor these scripts:

```
"test:unit": "...",
"test:integration": "...",
"test:acceptance": "npm run test:acceptance:setup && ..."
```

and then add all the suites in the main `test` command

```
"test": "npm run test:unit && npm run test:integration && npm run test:acceptance"
```